uvloop/_testbase.py

```diff
@@ -415,7 +415,9 @@ def __init__(self, test, sock, prog, timeout):
     def run(self):
         try:
             self._prog(TestSocketWrapper(self._sock))
-        except Exception as ex:
+        except (KeyboardInterrupt, SystemExit):
+            raise
+        except BaseException as ex:
```

Comment:
Maybe do `except Exception, asyncio.CancelledError` instead.  Catching `BaseException` always makes me uneasy.

Reply:
Why? We handle `KeyboardInterrupt` and `SystemExit`. Besides, that's how we do it in asyncio now, so there's no option, really :(